esp32/sdkconfig.h

```diff
@@ -36,7 +44,6 @@
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK 0
 #define CONFIG_TASK_WDT_CHECK_IDLE_TASK_CPU1 0
 
-#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
Compiling the example get-started/hello_world with FreeRTOS/UNICORE set to "yes" succeeds.  Try adding `-Wno-error=array-bounds` to CFLAGS for the dport_access.c file.